File: src/backend/app/Models/Payment.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Payment extends Model
{
    protected $table = 'payment';
    
    protected $primaryKey = 'paymentId';
    
    public $timestamps = false;

    protected $fillable = [
        'bookingId',
        'dateAndTime'
    ];

    public function booking()
    {
        return $this->belongsTo(Booking::class);
    }
}

File: src/backend/src/api/Models/Payment.cs
namespace Restify.API.Models;

public class Payment
{
	public DateTime			DateAndTime { get; set; }
	public decimal			Amount { get; set; }
	public int?				BookingId { get; set; }
	public int				Id { get; set; }
	public int				PaymentInformationId { get; set; }
}
